### ISSUE TITLE
Add an -AsArray switch to ConvertTo-Json to support unconditional conversion to a JSON array

### ISSUE THREAD

**Issue by mklement0** [PowerShell/PowerShell#6327]
Based on a [suggestion by @AikenBM](https://github.com/PowerShell/PowerShell/issues/3424#issuecomment-370918077), welcomed by @SteveL-MSFT:

Similar to how `@(...)` ensures treatment of any value as an array, it would be handy if an `-AsArray` switch could instruct `ConvertTo-Json` to treat its input as an [ultimately JSON] array, so you can _guarantee that your pipeline input is treated as an array_, even if it happens to be a single item only:

**Wishful thinking:**

```powershell
> $in = 1; $in | ConvertTo-Json -AsArray
[ 
  1 
] 
# scalar 1 was treated as an array

> $in = 1, 2; $in | ConvertTo-Json -AsArray
[ 
  1, 
  2 
]  
# Input that already is an array is left as-is
```

**Awkward workaround**, in the absence of `-AsArray`:

```powershell
> $in = 1; , @($in) | ConvertTo-Json -AsArray
[ 
  1 
] 
```

Environment data
----------------

Written as of PowerShell Core v6.0.1

**Comment by kalgiz** [comment]
Fix proposal in #6438.

**Comment by kalgiz** [comment]
Fixed in  #6438.
